src/main/java/no/unit/alma/bibs/AlmaItemsServiceImplementation.java

```diff
@@ -163,9 +169,12 @@ public ItemLoan createUserLoanOnItem(final String barcode, final String user_Id,
     }
 
     @Override
-    public ItemLoan updateUserLoanAndChangeDueDate(final String userId, final String loanId, final Calendar dueDate) {
+    public ItemLoan updateUserLoanAndChangeDueDate(final String userId, final String loanId, final Calendar dueDate) throws DatatypeConfigurationException {
         final ItemLoan itemLoan = new ItemLoan();
-        itemLoan.setDueDate(dueDate);
+        GregorianCalendar gregory = new GregorianCalendar();
```

Comment:
Føler at du kunne ha gitt denne navnet gregor :D

Reply:
It is an honour. Det finnes bare få folk som kan bære dette navn med stolthet.